ext/lib/crypto/mbedtls/Kconfig.tls-generic

```diff
@@ -68,7 +68,7 @@ config MBEDTLS_KEY_EXCHANGE_RSA_PSK_ENABLED
 
 config MBEDTLS_KEY_EXCHANGE_RSA_ENABLED
 	bool "Enable the RSA-only based ciphersuite modes"
-	default y
+	default y if !NET_L2_OPENTHREAD
```

Comment:
@nashif @jukkar Any opinion on default algorithms enabled in `Kconfig.tls-generic`? See the PR description for more details.